### ISSUE TITLE
[FEATURE] Meaningful Space: Add the option to fix elements

### ISSUE THREAD

**Issue by veradrawer** [Sanqui/obsidian-persistent-graph#1]
This is awesome, but It's halfway there.
If we could drag and lock certain nodes in space, we would have the first real map of content, not the dynamic diagram the graph currently is.

![imagen](https://user-images.githubusercontent.com/75331117/150645170-81357ab7-2762-440c-8ccb-9be043234b71.png)
As cool as Obsidian can distribute nodes, we lack the ability to give space meaning.

**Comment by Sanqui** [comment]
Locking nodes in space is actually possible (and I could persist locked nodes in saves).  I'm not sure how ergonomic I could make the user interface for this however.

**Comment by veradrawer** [comment]
I agree. The current implementation is that when you clic on a node, you can either enter the link it has, or you cna move it around.
The first thing that came to mind was a toggleable mode where clicking alternates node locking on space.
One clic, fixed, another clic, autospaced.

**Comment by CarlVitasa** [comment]
+! would be an awesome feature!
